### PR TITLE
1636 BaseTableAssert fix

### PR DESCRIPTION
**Pull request by ArKhmeln**

### CODE REVIEW

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java [hunk 1]
```diff
@@ -41,13 +41,32 @@ public A notEmpty() {
...
+        return (A) this;
+    }
+
+    /**
+     * Match passed value with table count
+     * @param count to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(int count) {
+        return count(Matchers.is(count));
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableExampleTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435&bid=16021372)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableExampleTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435&bid=16021372)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java (hunk 1 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435)

**Comment by mf-code** [comment]
[4473] - https://transfer.sh/zzijC/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4473] - https://transfer.sh/U5ydX/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4473] - https://transfer.sh/rBD66/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4473] - Allure report on Netlify: https://5e1de0dc455e480652f729a2--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4475] - https://transfer.sh/VmNuH/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4475] - https://transfer.sh/Scg5V/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4475] - https://transfer.sh/aLo5B/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4475] - Allure report on Netlify: https://5e1de8bed34f62beea9341a0--musing-swartz-6fc060.netlify.com

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java (hunk 1 above)
Comment (dexguitar) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Remove the unnecessary **'Matchers.'** part

Comment (dexguitar) [reply jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Okay, that's how it's supposed to be at this point. Please ignore.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableExampleTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435&bid=16021372)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java [hunk 2]
```diff
@@ -42,17 +42,36 @@ public A notEmpty() {
...
+        return (A) this;
+    }
+
+    /**
+     * Match passed value with table rows count
+     * @param count to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(int count) {
+        return count(Matchers.is(count));
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435)

**Comment by mf-code** [comment]
[4486] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4486] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4486] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4486] - Allure report on Netlify: https://5e1f008115dc0ab41a51b167--musing-swartz-6fc060.netlify.com

**Comment by dexguitar** [comment]
Please fix DataTableTests class failure

File: jdi-performance/src/test/java/org/mytests/tests/example/DataTableTests.java [hunk 3]
```diff
@@ -82,9 +82,9 @@ public void allDataFilterTest() {
     @Test
```
Comment (dexguitar) [inline jdi-performance/src/test/java/org/mytests/tests/example/DataTableTests.java]: The whole class fails on **setUp()**

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-html-tests/src/test/java/io/github/epam/example/complex/TableExampleTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435&bid=16021372)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java (hunk 2 above)
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4818435)

**Comment by mf-code** [comment]
[4489] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4489] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4489] - Could not save metadata Could not save metadata Could not save metadata

**Comment by mf-code** [comment]
[4489] - Allure report on Netlify: https://5e1f134cb38f919f3be6fd04--musing-swartz-6fc060.netlify.com

**Pull request merged**